xray/segment.go

```diff
@@ -52,6 +52,19 @@ func BeginFacadeSegment(ctx context.Context, name string, h *header.Header) (con
 	return context.WithValue(ctx, ContextKey, seg), seg
 }
 
+// BeginDefaultSanitizing returns sanitized string for a given san(SQL, HTTP, AWS, Metadata, Annotations).
+func BeginDefaultSanitizing(san string, value string) string {
```

Comment:
Does customer have to explicitly call `BeginDefaultSanitizing` before he adds the data? Can we do this for him?